jax/experimental/ode.py

```diff
@@ -77,6 +77,11 @@ def initial_step_size(fun, t0, y0, order, rtol, atol, f0):
 
   h0 = jnp.where((d0 < 1e-5) | (d1 < 1e-5), 1e-6, 0.01 * d0 / d1)
 
+  # This deviates slightly from Hairer and Wanner, but protects us from
+  # overshooting the goal time step. `h0` is especially radical in stiff
+  # systems.
+  h0 = jnp.clip(h0, 0.0, t1 - t0)
```

Comment:
If times are decreasing, then I guess things will already break quite badly later. Maybe better to stick with just `maximum` ?
```suggestion
  h0 = jnp.maximum(h0, t1 - t0)
```

Reply:
I think this should be minimum?